databroker/tests/test_broker.py

```diff
@@ -64,7 +64,7 @@ def test_uid_list_multiple_headers(db, RE):
     RE.subscribe(db.insert)
     uids = RE(pchain(count([det]), count([det])))
     headers = db[uids]
-    assert uids == [h['start']['uid'] for h in headers]
+    assert uids == tuple([h['start']['uid'] for h in headers])
```

Comment:
I think this change would make the reader wonder why `tuple()` is needed to make the test pass. In fact, of course, it is not needed: lists and tuples may be compared.

Reply:
Please tell me more about this magical comparison.
https://travis-ci.org/NSLS-II/databroker/jobs/269221759#L1455

Reply:
:sheep:. Indeed:

```python
>>> >>> (1,2) == [1,2]
False
```